compiler/rustc_borrowck/src/diagnostics/mutability_errors.rs

```diff
@@ -710,6 +705,63 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, 'tcx> {
         )
     }
 
+    fn construct_suggestion(&self, err: &mut DiagnosticBuilder<'_, ErrorGuaranteed>, local: Local) {
```

Comment:
> It might be worth giving this a more descriptive name - I'm sure that there are lots of suggestions being constructed during borrowck, so we mention the specific suggestion being constructed.

done